sqlstore/src/main/resources/db/migration/V8900__hibernatize_everything.sql

```diff
@@ -1,3 +1,19 @@
 UPDATE KitDescriptor SET kitType = UPPER(kitType), platformType = UPPER(platformType);
 
-UPDATE Platform SET name = UPPER(name);
\ No newline at end of file
+UPDATE Platform SET name = UPPER(name);
+
+CREATE TABLE ProjectOverview_Sample (
+  projectOverview_overviewId bigint(20) NOT NULL,
+  sample_sampleId bigint(20) NOT NULL,
+  CONSTRAINT projectOverview_sample_projectOverview_overviewId FOREIGN KEY (projectOverview_overviewId) REFERENCES ProjectOverview(overviewId),
+  CONSTRAINT projectOverview_sample_sample_sampleId FOREIGN KEY (sample_sampleId) REFERENCES Sample(sampleId)
+) ENGINE=InnoDB;
+
+INSERT INTO ProjectOverview_Sample(projectOverview_overviewId, sample_sampleId)
+  SELECT parentId, entityId FROM EntityGroup JOIN EntityGroup_Elements ON EntityGroup.entityGroupId = EntityGroup_Elements.entityGroup_entityGroupId
+   WHERE entityType = 'uk.ac.bbsrc.tgac.miso.core.data.Sample' AND parentType = 'uk.ac.bbsrc.tgac.miso.core.data.impl.ProjectOverview';
+
+DROP TABLE EntityGroup_Elements;
+DROP TABLE EntityGroup;
+
+UPDATE Pool_Elements SET elementType = CASE elementType WHEN 'uk.ac.bbsrc.tgac.miso.core.data.impl.LibraryDilution' THEN 'L' WHEN 'uk.ac.bbsrc.tgac.miso.core.data.impl.emPCRDilution' THEN 'E' END;
```

Comment:
We want to make Libraries Poolable in the future I think... 'L' would be confusing in that case. Use longer String identifiers?